dashboards/gateway-api-inference-extension/inference-extension-prometheus.json

```diff
@@ -0,0 +1,763 @@
+{
+    "displayName": "Inference Extension Prometheus Overview",
+    "dashboardFilters": [],
+    "labels": {
+      "cloud-monitoring-dashboard-importer": "",
+      "goog-imported-grafana-dashboard": "",
+      "goog-imported-grafana-dashboard-from-ui": "",
+      "goog-imported-grafana-revision": "16",
+      "goog-imported-via-script": ""
+    },
+    "mosaicLayout": {
+      "columns": 48,
+      "tiles": [
+        {
+          "height": 4,
+          "width": 39,
+          "widget": {
+            "text": {
+              "content": "### Please see https://github.com/kubernetes-sigs/gateway-api-inference-extension/blob/main/site-src/guides/metrics.md for more details of underlying metrics used in the dashboard.",
```

Comment:
You can use markdown syntax for a hyperlink instead of just the URL

Reply:
Done.